src/index.js

```diff
@@ -17,4 +18,8 @@ if (module.hot) {
   module.hot.accept('./components/app', render);
 }
 
+if (process.env.NODE_ENV === 'development') {
```

Comment:
`process.env` will be replaced with strings thanks to `webpack.EnvironmentPlugin`